src/rustllvm/PassWrapper.cpp

```diff
@@ -419,6 +420,10 @@ extern "C" LLVMTargetMachineRef LLVMRustCreateTargetMachine(
   Options.MCOptions.PreserveAsmComments = AsmComments;
   Options.MCOptions.ABIName = ABIStr;
 
+  if (LinkerIsLLD) {
+    Options.RelaxELFRelocations = true;
```

Comment:
Testing the effects of this is difficult, but I've filed https://github.com/fortanix/rust-sgx/issues/202 to track possible future improvements.